cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -197,7 +198,12 @@ def set_agent_creds(appliance, request, provider):
 def local_setup_provider(request, setup_provider_modscope, provider, appliance):
     # TODO: allow for vddk parameterization
     if provider.one_of(VMwareProvider):
-        appliance.install_vddk()
+        vddk_url = conf.cfme_data.get("basic_info").get("vddk_url").get('v6_0', None)
```

Comment:
Not actually fixed here - you have to give defaults at each step. Re-read my original comment.

```
In [1]: cfme_data = {'test': 'data'}

In [2]: cfme_data.get('junk').get('next', None)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-c62bd8e7091f> in <module>()
----> 1 cfme_data.get('junk').get('next', None)

AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
oops